mkt/ratings/serializers.py

```diff
@@ -82,6 +90,11 @@ def validate(self, attrs):
             user = self.request.user
             attrs['user'] = user
             attrs['ip_address'] = self.request.META.get('REMOTE_ADDR', '')
+            guessed_lang = guess_language(attrs['body'])
```

Comment:
Would we ever want `guessed_lang` to be falsey? We shouldn't need to explicitly check against `None`, right? I think an `or` would be more clear here.

```py
attrs['lang'] = guess_language(attrs['body']) or user.lang